Orange/tests/test_classification.py

```diff
@@ -190,6 +193,76 @@ def test_incompatible_domain(self):
         with self.assertRaises(DomainTransformationError):
             clf(titanic)
 
+    LEARNERS_CLASSIFICATION = [
```

Comment:
There is a function `all_learners()` in `LearnerAccessibility` class. Could that be used for testing instead of hard coding the `learners` list. If we add a new learner, we will most likely forget to add it to the list.